composer.json

```diff
@@ -20,14 +20,14 @@
     "require": {
         "php": "^7.1",
         "imagine/imagine": "^0.7.1|^1.1",
-        "symfony/asset": "^3.4|^4.2",
-        "symfony/filesystem": "^3.4|^4.2",
-        "symfony/finder": "^3.4|^4.2",
-        "symfony/framework-bundle": "^3.4|^4.2",
-        "symfony/options-resolver": "^3.4|^4.2",
-        "symfony/process": "^3.4|^4.2",
-        "symfony/templating": "^3.4|^4.2",
-        "symfony/translation": "^3.4|^4.2"
+        "symfony/asset": "^3.4|^4.2|^5.0",
+        "symfony/filesystem": "^3.4|^4.2|^5.0",
+        "symfony/finder": "^3.4|^4.2|^5.0",
+        "symfony/framework-bundle": "^3.4|^4.2|^5.0",
+        "symfony/options-resolver": "^3.4|^4.2|^5.0",
+        "symfony/process": "^3.4|^4.2|^5.0",
+        "symfony/templating": "^3.4|^4.2|^5.0",
+        "symfony/translation": "^3.4|^4.2|^5.0"
```

Comment:
mmmm I think the bundle doesn't use any of `"symfony/translation"` and `"symfony/asset"`